src/main/java/teammates/storage/api/FeedbackResponseMonitorDb.java

```diff
@@ -45,7 +45,7 @@ FeedbackResponseRecordAttributes makeAttributes(FeedbackResponseRecord entity) {
      * Gets a set of response records with chosen duration and interval.
      */
     public List<FeedbackResponseRecordAttributes> getResponseRecords(long duration, long interval) {
-        long startTimeInMilisec = System.currentTimeMillis() - duration;
+        long startTimeInMilisec = System.currentTimeMillis() - duration * 1000;
```

Comment:
Now that you have done it... Can frontend send in milliseconds instead? This looks very out of place.